esbuild.config.mjs

```diff
@@ -0,0 +1,40 @@
+/**
+ * Run with node esbuild.config.mjs in order to build the backend
+ */
+import * as esbuild from "esbuild";
+import start from '@es-exec/esbuild-plugin-start';
+
+// "watch" or "build"
+const command = process.argv[2];
+
+const baseOptions = {
+  entryPoints: [
+    'src/server.ts',
+    'config/**/*.ts',
+    'config/**/*.js',
+    'modules/server/**/*.ts',
+    'modules/server/**/*.js',
+    'modules/shared/**/*.ts',
+    'modules/shared/**/*.js',
```

Comment:
Not sure this is the correct thing to do. Adding just the server.ts entrypoint doesn't seem to pull in everything else.

I didn't add any of the html files...honestly I'm not sure how it works. I suspect that it is actually reading from the original un-built folder structure...?

eg, `res.render("modules/server/core/views/index"` might be looking at paths relative to WORKDIR.

We should probably fix it... but maybe in a follow-up PR?

Reply:
i imagine it's because our server code uses a lot of dynamic `require(...)` inside the code. however, for esbuild to correctly pull in all the relevant files, we need to be using es6 imports probably.

i think the implementation you have as is is also totally fine though

Reply:
Yeah I think you're right. But we're not really going to be able to get rid of the `require`s easily. Maybe we can try and typescriptify the whole thing in one go...